### PR TITLE
Error analysis - Sentry Quickstart

### LINKED ISSUE #31: Error Resolution Optimization Stack
### Optimizing Error Resolution Processes with Sentry Insights.

Extract and analyze error data from Sentry using Airbyte and dbt to optimize error resolution processes, reduce downtime, and improve user experience.

How to get started: 

- Check out the [README](https://github.com/airbytehq/quickstarts/tree/main) and the [Contribution instructions](https://github.com/airbytehq/quickstarts/blob/main/CONTRIBUTING.md) in this repo.
- Check one of the [existing quickstarts](https://github.com/airbytehq/quickstarts#list-of-available-quickstarts) and observe their structure.
- (Optional, but highly recommended) Implement one of the quickstarts by following the instructions in their README. I recommend starting with [E-commerce analytics in BigQuery](https://github.com/airbytehq/quickstarts/tree/main/ecommerce_analytics_bigquery)

### PR DESCRIPTION
**Pull request by mit-27**
Closes #31 

- Set up the Error analysis stack quickstart using Sentry, Airbyte, dbt and dagster.

Please inform me if any changes are needed.

### CODE REVIEW

File: error_analysis_stack_sentry/orchestration/orchestration/assets.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+airbyte_instance = AirbyteResource(
+    host="localhost",
+    port="8000",
+    # If using basic auth, include username and password:
+    username="airbyte",
+    password=os.getenv("AIRBYTE_PASSWORD")
+)
+
+airbyte_assets = load_assets_from_airbyte_instance(airbyte_instance, key_prefix="faker")
```
Comment (ThaliaBarrera) [inline error_analysis_stack_sentry/orchestration/orchestration/assets.py:21]: The `key_prefix` should be updated here to indicate your actual source. I don't see you have a source "faker" in your project.

File: error_analysis_stack_sentry/assets/orchestration_lineage.png
[the remark this answers is not in the document]
Comment (mit-27) [reply error_analysis_stack_sentry/assets/orchestration_lineage.png:1]: Sure, I will make these changes.

**Comment by mit-27** [comment]
I corrected DAG and added Airbyte as source to dbt. Could you @ThaliaBarrera review it ?

**Review by ThaliaBarrera — APPROVED** [review]
Looks good now! Thanks @mit-27

**Pull request merged**